website/addons/github/routes.py

```diff
@@ -4,134 +4,118 @@
 
 from website.addons.github import views
 
-settings_routes = {
+
```

Comment:
For sure. I made a note of that during the dev meeting on Tuesday but it slipped my mind.